src/compiler/scanner.ts

```diff
@@ -364,6 +365,11 @@ namespace ts {
 
     const hasOwnProperty = Object.prototype.hasOwnProperty;
 
+    export function isWhiteSpaceLike(ch: number): boolean {
+        return isWhiteSpace(ch) || isLineBreak(ch);
+    }
+
+    /** Does not include line breaks. For that, see isWhiteSpaceLike. */

```

Comment:
The regex terminology for including newlines seems to be "multiline". However, I would normally expect "whitespace" to include newlines. So maybe there should be `isWhiteSpace`, `isWhiteSpaceSingleLine`, and `isLineBreak`?